betterproto/__init__.py

```diff
@@ -741,10 +722,10 @@ def parse(self: T, data: bytes) -> T:
                 pos = 0
                 value = []
                 while pos < len(parsed.value):
-                    if meta.proto_type in ["float", "fixed32", "sfixed32"]:
```

Comment:
Great, thank you!

Since you had another PR with these changes, we can merge that one first, if you wouldn't mind to resolve the merge conflicts there.